app/Controllers/importExportController.php

```diff
@@ -798,135 +728,113 @@ public function exportFile($username, $export_opml = true, $export_starred = fal
 	 * Parameters are:
 	 *   - export_opml (default: false)
 	 *   - export_starred (default: false)
+	 *   - export_labelled (default: false)
 	 *   - export_feeds (default: array()) a list of feed ids
 	 */
 	public function exportAction() {
 		if (!Minz_Request::isPost()) {
-			Minz_Request::forward(array('c' => 'importExport', 'a' => 'index'), true);
-		}
-		$this->view->_layout(false);
-
-		$nb_files = 0;
-		try {
-			$nb_files = $this->exportFile(
-				Minz_Session::param('currentUser'),
-				Minz_Request::param('export_opml', false),
-				Minz_Request::param('export_starred', false),
-				Minz_Request::param('export_labelled', false),
-				Minz_Request::param('export_feeds', array())
+			return Minz_Request::forward(
```

Comment:
I just remembered I forgot to explain these `return`. It's actually useless because the `forward(..., true)` will immediately trigger a redirection, but it's implicit and not clear for a new developer. I suggest we start to put explicit `return` when we want the method to stop, what do you think?